Custom-Audio-Driver/Custom-Audio-Driver/DefaultAudioDevice.swift

```diff
@@ -516,10 +516,10 @@ extension DefaultAudioDevice {
         do {
             try session.setPreferredSampleRate(Double(DefaultAudioDevice.kSampleRate))
             try session.setPreferredIOBufferDuration(0.01)
-            let audioOptions = AVAudioSessionCategoryOptions.mixWithOthers.rawValue |
-            AVAudioSessionCategoryOptions.allowBluetooth.rawValue |
-            AVAudioSessionCategoryOptions.defaultToSpeaker.rawValue
-            try session.setCategory(AVAudioSessionCategoryPlayAndRecord, mode: AVAudioSessionModeVideoChat, options: AVAudioSessionCategoryOptions(rawValue: audioOptions))
+            let audioOptions = AVAudioSession.CategoryOptions.mixWithOthers.rawValue |
```

Comment:
There are some nicer ways to do this in Swift rather than bits magic ;)  eg: array of enum values